src/view/ChewingEditor.cpp

```diff
@@ -190,11 +199,16 @@ void ChewingEditor::setupFileSelection()
 
 void ChewingEditor::setupImport()
 {
+    // initialize importCounter_
+    importedCounter_ = 0;
+
     shortcut_import_ = new QShortcut(Qt::CTRL + Qt::Key_I, this);
     connect(
         shortcut_import_, SIGNAL(activated()),
         this, SLOT(selectImportFile())
     );
+    // initialize importCounter_
+    importedCounter_ = 0;
```

Comment:
Sorry but I wasn't so sure about why you mean by inconsistent naming, can you give an example of consistent naming, according to this situation?
I named it this way according to how other variable are named in `ChewingEditor.h`.